inference.Rmd

```diff
@@ -702,6 +730,7 @@ one_sample <- one_sample |>
 one_sample
 one_sample_dist <- ggplot(one_sample, aes(price)) +
```

Comment:
need to fix the figure formatting here 
<img width="423" alt="Screen Shot 2021-09-26 at 9 15 58 PM" src="https://user-images.githubusercontent.com/3838091/134844693-07bd4ec7-2f76-4800-956c-37ab20864d82.png">

Reply:
created new issue for it